apache2/msc_reqbody.c

```diff
@@ -607,6 +607,19 @@ apr_status_t modsecurity_request_body_end(modsec_rec *msr, char **error_msg) {
     /* Note that we've read the body. */
     msr->msc_reqbody_read = 1;
 
+
+    /* Check that we are not over the request body no files limit. */
+    if (msr->msc_reqbody_no_files_length >= (unsigned long)msr->txcfg->reqbody_no_files_limit) {
+        *error_msg = apr_psprintf(msr->mp, "Request body no files data length is larger than the "
+            "configured limit (%ld).", msr->txcfg->reqbody_no_files_limit);
+        if (msr->txcfg->debuglog_level >= 1) {
+            msr_log(msr, 1, "%s", *error_msg);
+        }
+
+        return -5;
```

Comment:
what is -5? Does this value get printed in log?

Reply:
This means body was too long. They return this already another place: https://github.com/Microsoft/ModSecurity/blob/b2ec8ab5d0ebb0f91a5f32636427e0be731f5660/apache2/msc_reqbody.c#L408
and they catch it here:
https://github.com/Microsoft/ModSecurity/blob/b2ec8ab5d0ebb0f91a5f32636427e0be731f5660/apache2/apache2_io.c#L299